### DISCUSSION TITLE
Make Backgrounds seamlessly fading into each other

### DISCUSSION THREAD

**Discussion by smxko** [t1m0thyj/WinDynamicDesktop#366] [Ideas]
This app has so much potential.
Why is there not option to create "in between" steps between the wallpapers based on transparency levels of two backgrounds? For example, instead of hard switching from one wallpaper to another, just make them blend together smoothly. This would work perfectly for backgrounds like the Mojave Desert.

Let's say there is a time span of one hour. At the beginning, Wallpaper_01 is at 100% opacity. After 15 minutes, Wallpaper_01 is at 75% opacity and the underlaying Wallpaper _02 slowly blends in.
30 minutes: Wallpaper_1 at 50%, as a logical consequence Wallpaper_02 will be 50% visible
After one hour (after the app would switch the wallpaper with a hard cut) Wallpaper_02 has seamlessly blended into the initial canvas of Wallpaper_01.
So you just decrease the opacity of the background, increasing the transparency over time.

Of course you could generate the opacity values dynamically per minute, otherwise you'd end up with the same hard transitions as of now.

Here's a video I found for reference: https://www.youtube.com/watch?v=HFphX4dpAHk

**Comment by MidnightProwler** [comment]
Thts actually kinda nice, i'd like to see it implemented

**Comment by emberaholdon** [comment]
> This would work perfectly for backgrounds like the Mojave Desert.

No. It would actually look horrible. Just try it yourself in paint.net or another image editor. It looks like a rendering issue caused by faulty GPU to be honest.

The Mojave theme has 16 images for the whole day, I wouldn’t call that “hard switching”.

What you want is a 24 hour live video of the area. :)

**Reply by smxko** [reply]
> What you want is a 24 hour live video of the area. :)

I just wanted to make it look like on Mac OS :|

**Reply by smxko** [reply]
> No. It would actually look horrible. Just try it yourself in paint.net or another image editor. It looks like a rendering issue caused by faulty GPU to be honest.

It looks dope. Just copy two of the key images (evening / night) into Photoshop. However, bottom layer must be at 100% opacity, otherwise it will not work. Dragging around the opacity of the top layer you can makes it look like a timelapse.

**Reply by emberaholdon** [reply]
> > What you want is a 24 hour live video of the area. :)
> 
> I just wanted to make it look like on Mac OS :|

I’n not sure macOS transitions the images like a timelapse. See: https://youtu.be/qZyGKIBylHs (8:15)
